tools/install.sh

```diff
@@ -94,5 +96,5 @@ if [ $DRY_RUN -eq 1 ]; then
     echo
     echo "$VAR"
 else
-    $VAR
+    $VAR | (echo && echo && echo "$VAR" && echo)
```

Comment:
what does that `| (echo && echo && echo "$VAR" && echo)` supposed to do? 🤔

Reply:
each echo prints new line - so it is for better visual in console

Reply:
why is `|` there though? 🤔
also, if you just need line breaks - wouldn't it be easier to do `"\n\n$VAR"`? 🤔

Reply:
`"\n\n$VAR"` for some reason did not worked for me. 
About `|` - I added this that in case of failure the $VAR wouldb eprinted also. That you could debug runned command.

Reply:
I think you need to use `printf` instead of `echo` for `\n` to work